charts/ccsm-helm/charts/identity/templates/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "identity.fullname" . }}
+  labels: {{- include "identity.labels" . | nindent 4 }}
+  annotations: {{- toYaml  .Values.global.annotations | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "identity.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels: {{- include "identity.labels" . | nindent 8 }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        {{- if .Values.image.tag }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        {{- else }}
+        image: "{{ .Values.image.repository }}:{{ .Values.global.image.tag }}"
+        {{- end }}
+        env:
+          - name: SERVER_PORT
+            value: "8080"
```

Comment:
`8080` is the default value, so this setting can also be removed.

Reply:
Right now yes, but better to be explicit here. What if you change the defaults :sweat_smile: I plan to also introduce a property in the charts, than this can be overwritten.